src/ch03/p1_digram_counter.py

```diff
@@ -93,7 +93,7 @@ def main():
     print(f'Analyzing: {word}\n')
     print('In the Ubuntu default `american-english` dictionary, these are\n'
           'the digram counts for the above word:')
-    digram_count = digram_counter(word)
+    digram_count = digram_counter(word, DICTIONARY_FILE_PATH)
```

Comment:
The DICTIONARY_FILE_PATH cannot be changed on the module level because it is defined once the module is imported. However, if it is used inside a function, it can be patched on the function level.